### PR TITLE
NNS1-3485: initial setup for the new reporting page

### PR DESCRIPTION
**Pull request by yhabib**
# Motivation

This new page will display all reporting functionalities currently found in the `AccountMenu`. 
This first iteration introduces the page without any logic. Subsequent pull requests will migrate the Export components from the `AccountMenu` to this page.

# Changes

- New Reporting page

# Tests

- Unit tests for the route's layout and page
- Unit tests for the page

# Todos

- [ ] Add entry to changelog (if necessary).
Not yet

Next PR: #5965

### CODE REVIEW

File: frontend/src/routes/(app)/(nns)/reporting/+layout.svelte [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
```
Comment (yhabib) [inline frontend/src/routes/(app)/(nns)/reporting/+layout.svelte:1]: This is a copy/paste from the Settings app.

File: frontend/src/lib/pages/SignInReporting.svelte [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  import EmptyCards from "$lib/components/common/EmptyCards.svelte";
+  import SignIn from "$lib/components/common/SignIn.svelte";
+  import { i18n } from "$lib/stores/i18n";
+  import { IconSettingsPage, PageBanner } from "@dfinity/gix-components";
+</script>
+
+<main>
+  <div class="content">
+    <PageBanner>
+      <IconSettingsPage slot="image" />
```
Comment (yhabib) [inline frontend/src/lib/pages/SignInReporting.svelte:11]: I'm reusing this icon for the time being. We might have a new one in the future but it is not confirmed yet.

File: frontend/src/routes/(app)/(nns)/reporting/+layout.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/routes/(app)/(nns)/reporting/+layout.svelte:1]: How about something like 
```
onMount({
  if (feature_flag === false) await goto(AppPath.Accounts);
});
```
just to stay on the safe side.

**Review by mstrasinskis — APPROVED** [review]
LGTM, thank you. Just an optional comment.

**Pull request merged**